build/constants.yaml

```diff
@@ -324,6 +324,7 @@
     testing-sdk-storage-url-key-name: ORYX_TEST_SDK_STORAGE_URL
     dev-sdk-storage-base-url: https://oryxsdksdev.blob.core.windows.net
     sandbox-sdk-storage-base-url: https://oryxsdkssandbox.blob.core.windows.net
+    private-sdk-storage-base-url: https://oryxsdksprivate.blob.core.windows.net
```

Comment:
You'll want to run `generateConstants.sh` to propagate this for your pipeline script to use